docs/src/components/banner/AppFrameBanner.tsx

```diff
@@ -34,12 +34,17 @@ export default function AppFrameBanner() {
   const pageContext = React.useContext(PageContext);
   const productName = convertProductIdToName(pageContext) || 'MUI';
 
+  // Only show banner on MUI X product pages
+  if (productName !== 'MUI X') {
+    return null;
+  }
```

Comment:
New condition that should only show the survey on X docs. Though it doesn't work in the preview. I guess that because of the docs config?